app/src/main/java/fr/jaetan/jbudget/app/home/views/HomeScreen.kt

```diff
@@ -11,21 +10,22 @@ import fr.jaetan.jbudget.R
 import fr.jaetan.jbudget.app.home.HomeViewModel
 
 @OptIn(ExperimentalMaterial3Api::class)
-@SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
+//@SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
 @Composable
 fun HomeScreen(navController: NavHostController, ) {
-    val viewModel = HomeViewModel()
-    Scaffold( content = {HomeContent()}, floatingActionButton = {HomeFAB(viewModel)}, topBar = { AppBar() })
+    val viewModel = HomeViewModel(navController)
+    Scaffold( content = {HomeContent(it)}, floatingActionButton = {HomeFAB(viewModel)}, topBar = { AppBar(navController) })
 }
 
 
+
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun AppBar() {
+fun AppBar(navController: NavHostController) {
     TopAppBar(
         title = { Text(text = stringResource(R.string.app_name)) },
         actions = {
-            IconButton(onClick = { /*TODO*/ }) {
+            IconButton(onClick = {navController.navigate("settings")}) {
```

Comment:
Utilise `Screen.Settings.route` plutôt que de marqué la route en dur ici

Reply:
Noté, prochain commit c'est corrigé

Reply:
D'acc parfait